### ISSUE TITLE
prior train

### ISSUE THREAD

**Issue by yangzhip** [lucidrains/DALLE2-pytorch#236]
RuntimeError: Sizes of tensors must match except in dimension 1. Expected size 768 but got size 512 for tensor number 3 in the list.

my clip feature is 512 how can i train this prior

**Comment by lucidrains** [comment]
@yangzhip you need to set `image_embed_dim = 512` when instantiating `DiffusionPrior`

**Comment by yangzhip** [comment]
> @yangzhip you need to set `image_embed_dim = 512` when instantiating `DiffusionPrior`

Yes, the dimension I have changed in the config file is 512, but there is still an error. The character of the text in the code after the clip is 768. How to change it here？ Thank you!

**Comment by lucidrains** [comment]
@yangzhip could you share your script?
